backend/api/tests/schema/test_newsletter.py

```diff
@@ -3,11 +3,11 @@
 
 from newsletters.models import Subscription
 
-pytestmark = mark.skip
+# pytestmark = mark.skip
 
 
-@mark.django_db
-def test_subscribe_to_newsletter(graphql_client):
+def test_subscribe_to_newsletter(graphql_client, mocker):
```

Comment:
I don't know why this test is failing: 

I have patched the method to return True, but seems like is calling the real API... I'm confused... 🤔 


```python
api/tests/schema/test_newsletter.py:31: AssertionError
------------------------------ Captured log call -------------------------------
ERROR    integrations.mailchimp:mailchimp.py:34 Error from mailchimp: {'type': 'mailchimp.com/developer/marketing/docs/errors', 'title': 'Forbidden', 'status': 403, 'detail': "The API key provided is linked to datacenter 'secret'", 'instance': 'e141fa98-4f5f-0e1a-e632-a064ab5a6535'}
ERROR    api.newsletters.forms:forms.py:21 403 Forbidden: The API key provided is linked to datacenter 'secret'
Traceback (most recent call last):
  File "/home/runner/work/pycon/pycon/backend/api/newsletters/forms.py", line 19, in save
    return subscribe(email)
  File "/home/runner/work/pycon/pycon/backend/integrations/mailchimp.py", line 35, in subscribe
    raise MailchimpError(
integrations.mailchimp.MailchimpError: 403 Forbidden: The API key provided is linked to datacenter 'secret'
```

Reply:
I think you have to do

```python
with mocker.patch("integrations.mailchimp.subscribe") as mock_subscription:
    mock_subscription.return_value = True
    # call the API inside the with block
```

otherwise right now it doesn't patch anything

Reply:
I tried also that but was not changing so much... I will give another shot...

Reply:
I also tested on docker locally and the test was working now... but in the CI it fails again... :(  
I don't understand what it's happening 🤔